src/components/events/LiveAlbumUploader.tsx

```
@@ -0,0 +1,100 @@
+      
+      const supabaseUrl = import.meta.env.VITE_SUPABASE_URL; // Assuming standard Vite env vars
+      const bucketName = "live-album-photos"; // Make sure to create this bucket in Supabase!
+
+      // 3. Upload using their utility
+      await uploadFileWithProgress(
+        supabaseUrl,
+        session.access_token,
+        bucketName,
+        filePath,
+        file,
+        (percent) => setProgress(percent)
```

ESLint

[error] Insert `,`

(prettier/prettier)